.github/workflows/ci.yml

```diff
@@ -18,9 +18,11 @@ jobs:
       - name: Setup python environment
         run: ./.github/scripts/setup_python_env.sh
         shell: bash
+      - name: Install working dir package
```

Comment:
`./.github/scripts/setup_python_env.sh` installs the package

Reply:
Without this there is a file path mismatch and 0% coverage score. Installing the package in the working directory using pip's editable mode avoids it.

Reply:
what is the recommended way for developers to install this package?

Reply:
I figured if you specify the package_name instead of path it works